### PR TITLE
* 1704 bug remove properties from kryptonrichttextbox v90

### LINKED ISSUE #1704: [Bug]: Remove properties from `KryptonRichtTextBox`
**Describe the bug**
KRTB State(*).Content objects have a property called: SynchronizeDropDownWidth.
This does not belong there and can be removed.

image

### PR DESCRIPTION
**Pull request by Wagnerp**
* Resolved #1704 

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs [hunk 1]
```diff
@@ -2442,10 +2442,11 @@ protected override void OnPaint(PaintEventArgs? e)
                 ForceControlLayout();
             }
 
-            if (StateCommon.ComboBox.Content.SynchronizeDropDownWidth)
-            {
-                DropDownWidth = Size.Width;
-            }
+            // ToDo: Create a new API for this in a later version
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs:2445]: Why, Where did the requirement for this come from?

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs:2445]: Through some testing, I noticed that the width of the KCB didn't affect the dropdown, which leads to a poor user experience.

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs:2445]: That's a bug in the KryptonComboBox on the Form (Regression!!!!)
BUT, the Ribbon works as expected.
image

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs:2445]: Would be useful to have a option to link  up the size width & drop-down width.

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs:2445]: #1706

**Pull request merged**